local-kubernetes/README.md

```diff
@@ -2,17 +2,49 @@
 
 This folder contains the scripts necessary to run the entire environment locally. 
 
-To run the system, you need to create a secret to access the private container registry.
+## Set Up Kubernetes
+
+To begin, make sure you have kubernetes running locally as per the [Kubernetes Dev Setup Instructions](../Docs/DevSetup/local-k8s-setup.md)
+
+Create the buying catalogue namespace - `kubectl apply -f local-namespace.yml`
+
+To run the system, you need to create a secret in kubernetes to access the private container registry, as per the [Connect Local Kubernetes with our Private Container Registry Instructions](../Docs/DevSetup/local/k8s-private-registry.md).
+
+You will also need a secret for the local instances of sql server - `kubectl create secret generic mssql --from-literal=SA_PASSWORD="<YOUR PASSWORD>" -n buyingcatalogue`
```

Comment:
Needs a note to explain minimum password length (8 characters according to SQL Server)

Reply:
added